docker-compose-native/docker-compose.yml

```diff
@@ -32,10 +40,11 @@ services:
     networks:
       presto:
         ipv4_address: 172.19.0.3
+
   coordinator:
     container_name: coordinator
-    image: prestodb/presto:0.282
-    platform: linux/amd64
+    image: prestodb/presto:latest
+    platform: linux/arm64
```

Comment:
You cannot remove amd64 support. You may add another compose file for arm64.

Reply:
Added another docker compose file, thanks.